payments-ui-core/src/main/java/com/stripe/android/ui/core/FieldValuesToParamsMapConverter.kt

```diff
@@ -52,6 +53,11 @@ class FieldValuesToParamsMapConverter {
                         blikCode
                     )
                 }
+            } else if (code == PaymentMethod.Type.Konbini.code) {
```

Comment:
This special casing kills me. But it seemed out of scope to refactor this while adding it. We should discuss what it would take to make this better as a follow up.